common.sh

```diff
@@ -428,7 +428,9 @@ if [[ ! -z ${AGENT_E2E_TEST_SCENARIO} ]]; then
           export MASTER_MEMORY=32768
           export NUM_WORKERS=0
           export NETWORK_TYPE="OVNKubernetes"
-          export AGENT_PLATFORM_TYPE="none"
+          if [[ "${AGENT_PLATFORM_TYPE:-baremetal}" != "external" ]]; then
+            export AGENT_PLATFORM_TYPE="none"
+          fi
```

Comment:
If the two accepted values for `AGENT_PLATFORM_TYPE` are just `none` and `external`, then I'd say that a validation rule could be more clear, informing also the user of the invalid value (keeping a default value of `none` if not set, as before)